nipype/interfaces/afni/preprocess.py

```diff
@@ -1802,28 +1802,107 @@ class QualityIndex(CommandLine):
 
 class ROIStatsInputSpec(CommandLineInputSpec):
     in_file = File(
-        desc='input file to 3dROIstats',
+        desc='input dataset',
         argstr='%s',
-        position=-1,
+        position=-2,
         mandatory=True,
         exists=True)
-    mask = File(desc='input mask', argstr='-mask %s', position=3, exists=True)
+    mask_file = File(desc='input mask', argstr='-mask %s', exists=True)
```

Comment:
When renaming a field, you should use `deprecated` and `new_name` (see the [developer guide](https://nipype.readthedocs.io/en/latest/devel/interface_specs.html)) fields (in addition to the new trait).

```Python
mask = File(desc='input mask', argstr='-mask %s', position=3, exists=True,
            deprecated='1.1.4', new_name='mask_file')
mask_file = File(desc='input mask', argstr='-mask %s', exists=True)
```